integration/failpoint/cmd/containerd-shim-runc-fp-v1/main.go

```diff
@@ -0,0 +1,32 @@
+//go:build linux
+// +build linux
```

Comment:
You do not need this if you change the file name to `main_linux.go`

Reply:
Updated.